keyboards/xd60/keymaps/styr/keymap.c

```diff
@@ -0,0 +1,62 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
+
+#define _BL 0
+#define _FL 1
+#define _LL 2
+// #define _RS 3
+
+#define _______ KC_TRNS
+#define XXXXX KC_NO
```

Comment:
If you make this 7 Xs, it's already included in `#include QMK_KEYBOARD_H`

Reply:
^